src/System.Net.Http/tests/FunctionalTests/MultiInterfaceNonRewindableReadOnlyStream.cs

```diff
@@ -0,0 +1,133 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Windows.Foundation;
+using Windows.Storage.Streams;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class MultiInterfaceNonRewindableReadOnlyStream : Stream, IInputStream
```

Comment:
It's called "multi interface" because it involves multiple interfaces from both WinRT stream types (IInputStream for example) and .NET stream types.